rag/svr/task_executor_refactor/dataset_wiki_generator.py

```
@@ -0,0 +1,778 @@
+    kb_id_str = str(ctx.kb_id)
+    entity_rows, relation_rows = build_wiki_page_graph(pages or [], kb_id_str)
+
+    index = search.index_name(ctx.tenant_id)
+
+    async def _replace_bucket(kwd: str, rows: List[Dict]) -> None:
+        try:
+            await thread_pool_exec(
+                settings.docStoreConn.delete,
+                {"compile_kwd": kwd}, index, ctx.kb_id,
+            )
+        except Exception:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)